code/addons/docs/src/preset.ts

```diff
@@ -147,6 +147,8 @@ export const viteFinal = async (config: any, options: Options) => {
       resolve: {
         alias: {
           react,
+          // Vite doesn't respect export maps when resolving a path, so we need to do that manually here
+          'react-dom/server': `${reactDom}/server.browser.js`,
```

Comment:
Does this entry point exist in React 16?

Reply:
Yes it does, but I'll make sure to test a React 16 sandbox too.